src/OrchardCore.Modules/OrchardCore.ReCaptcha/Views/Items/ReCaptchaPart.Fields.Edit.cshtml

```diff
@@ -1,16 +1,24 @@
 @using OrchardCore.ReCaptcha.Forms
 @model OrchardCore.ReCaptcha.Forms.ReCaptchaPartViewModel
+
+@{	
+    var settingsUrl = Url.Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = OrchardCore.ReCaptcha.Drivers.ReCaptchaSettingsDisplayDriver.GroupId });	
+    var hyperLinkAttributes = $"class=alert-link target=_blank href={settingsUrl}";	
+    var successMessage = T["<a {0}>ReCaptcha settings</a> are configured.", hyperLinkAttributes];	
+    var warningMessage = T["<a {0}>ReCaptcha settings</a> are not configured. The captcha will not be displayed.", hyperLinkAttributes];	
+}
+
 @{
     if (Model.SettingsAreConfigured)
     {
         <div class="alert alert-success">
-            <a asp-controller="Admin" asp-area="OrchardCore.Settings" asp-action="Index" asp-route-groupId="@OrchardCore.ReCaptcha.Drivers.ReCaptchaSettingsDisplayDriver.GroupId" class=alert-link target=_blank>@T["ReCaptcha settings"]</a> @T["are configured."]
+            @successMessage
```

Comment:
Can we use the `TagBuilder` for simplify and printout one div only

If you don't mine I can push a commit or merge it as its